### PR TITLE
Exceptions

### PR DESCRIPTION
**Pull request by drdovn**
change count-methods into get-methods
add exceptions for checking the existence of wanted transport and checking right type for the array length
fix some misprint

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -53,19 +71,39 @@ public static void main(String[] args) throws Exception {
...
             String name = in.next();
-            Transport yourTransport = Charlies.createTransport(name);
+
+            Transport yourTransport = new Transport();
+            try {
+                 yourTransport = Charlies.createTransport(name);
+            }catch(NoSuchTransport e){
+                e.printStackTrace();
+                System.exit(1);
+            }catch (CreatingProblems e) {
```
Comment (shukal94) [inline Main.java:81]: which  creating problems?

Comment (drdovn) [reply Main.java:81]: IllegalAccessException, InvocationTargetException, InstantiationException 
without handling these exceptions java doesn't allow to use .newInstance(), which creates an Object, so even if I don't know exactly how this problem can appear I decided to divide find-class-exception and create-object-exception

Comment (shukal94) [reply Main.java:81]: ObjectCreationException maybe better?

Comment (drdovn) [reply Main.java]: rename

**Pull request merged**